sections/03-methods.md

```diff
@@ -0,0 +1,65 @@
+
+#### Practice-Based Research and Adjacency
+
+The subjugation mode is clearly at play also in the increasingly-necessary construction of "practice-based research" in arts scholarship, in which this project ambivalently participates. According to proponents such as Graeme Sullivan, practice-based research asserts that a studio art practice can function as a form of scholarly research, and that "the studio experience is a form of intellectual and imaginative inquiry and is a site where research can be undertaken that is suf­ficiently robust to yield knowledge and understanding that is individually situated and socially and culturally relevant" (90). I do not entirely disagree, though there are fairly profound epistemological and artistic problems to be tangled with. The risk to art in this configuration is that it will always be subjugated to and justified by the regime of scholarly research, reduced, at worst, to a form of "empirical noodling."
+
+In response to this risk, UK composer John Croft argues explicitly that art and music have no role or function as research. He invokes Heidegger's position that, though art undoubtedly possesses "cognitive content," this material resists conceptualization. Art, in contrast to research, "_presents_ rather than _represents_, _discloses_ without describing" (8). The problem with a critique like this, of course, is that it limits art's potentiality by boxing it into a generalized, universal, vaguely transcendental definition, rather than seeing it a variable and living field of context-specific contingencies and forces. Instead, the question we should ask is: How is research knowledge "yielded" by studio practice, and is there perhaps a more complex network of influence and modulation at play than Sullivan or Croft are willing to acknowledge?
+
+One of the roles for disciplinary adjacency, in my mind, is to help prevent the subjugation of art by conventional research practices by highlighting, again, their operational closure. Art and research remain (potentially) distinct, yet still able to act as "lenses" that can transform and catalyze (but not necessarily cause) effects in each other<sup>4</sup> (Cecchetto, "Thoughts on Creative Practice and Research"). While any one of my videos are not, in themselves, research artifacts, the iterative and reflexive practice of making art (of being out in the world with my camera, or in the studio editing or programming) has powerful indirect influences via "systemic perturbation", helping me to cut new theoretical paths through conceptual thickets. "Any information value the perturbation takes on is constituted strictly by the distinctions belonging to the organization of the autopoietic machine itself" (Bryant 141). This means, I hope, that art and research can interact substantially with each other, without requiring them to be unified or linearly related, without having to share the same epistemic value system.
```

Comment:
This is very good, but could be tightened up a little, perhaps at the risk of exposing it to a systemic perturbation : P
Given "operational closure" is so crucial to many of these arguments we (or a hypothetical reader) should be clearer about what it is.
Also, the overall point doesn't necessarily falsify Croft's, which is interesting in itself